pyproject.toml

```diff
@@ -36,7 +36,7 @@ testing = [
     "pytest-ruff",
     "pytest-cov",
     "pytest-timeout",
-    "ruff==0.4.4",
+    "ruff>=0.4.4",
```

Comment:
Same as above comment: If it was pinned before, do we have any concerns about updating it now? If not, can we just remove the `>=0.4.4` part?

Reply:
Yeah we can get rid of it